.github/workflows/labels.yml

```diff
@@ -40,16 +45,26 @@ jobs:
       - name: Install dependencies
         run: npm install @actions/artifact bottleneck
 
+      # Use a GitHub App, because it has much higher rate limits: 12,500 instead of 5,000 req / hour.
+      - uses: actions/create-github-app-token@df432ceedc7162793a195dd1713ff69aefc7379e # v2.0.6
+        if: vars.NIXPKGS_CI_APP_ID
+        id: app-token
+        with:
+          app-id: ${{ vars.NIXPKGS_CI_APP_ID }}
+          private-key: ${{ secrets.NIXPKGS_CI_APP_PRIVATE_KEY }}
+          permission-pull-requests: write
```

Comment:
```suggestion
          permission-issues: write # needed to create *new* labels
          permission-pull-requests: write
```

Reply:
That said, maybe we should be explicit about this in a comment?

Either update the existing comment on the `GITHUB_TOKEN` permissions to mention it is mostly useful on forks, or add a comment to the app-token permissions saying why it isn't needed and/or that the app currently doesn't support it.